CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -0,0 +1,73 @@
+- Update Multi-cluster user guide to provide more details for Gateway enablement. ([#4889](https://github.com/antrea-io/antrea/pull/4889), [@luolanzone])
+- Update documentation for recent MetalLB versions. ([#4803](https://github.com/antrea-io/antrea/pull/4803), [@antoninbas])
+- Add support for short-circuiting in AntreaProxy to ensure that the traffic from Pod/Node clients to
+external addresses behaves the same way as the traffic from external clients to external addresses. ([#4815](https://github.com/antrea-io/antrea/pull/4815), [@hongliangl])
+
+### Fixed
+
+- Use LOCAL instead of CONTROLLER as the in_port of packet-out message to fix Windows agent crash issue. ([#4992](https://github.com/antrea-io/antrea/pull/4992), [@tnqn])
+- Fix the Antrea Agent crash issue on Windows by running modules that rely on Services after AntreaProxy is ready. ([#4946](https://github.com/antrea-io/antrea/pull/4946), [@tnqn])
+- Fix a bug that a deleted NetworkPolicy is still enforced when a new NetworkPolicy with the same name exists. ([#4986](https://github.com/antrea-io/antrea/pull/4986), [@tnqn])
+- Make FQDN NetworkPolicy work for upper case FQDNs. ([#4934](https://github.com/antrea-io/antrea/pull/4934), [@GraysonWu])
+- Fix a bug that K8s Networkpolicy audit logging doesn't work for Service access. ([#4780](https://github.com/antrea-io/antrea/pull/4780), [@qiyueyao])
+- Fix Service not being updated correctly when stickyMaxAgeSeconds or InternalTrafficPolicy is updated. ([#4845](https://github.com/antrea-io/antrea/pull/4845), [@tnqn])
+- Fix EndpointSlice API availablility check to resolve the issue that AntreaProxy always falls back to the Endpoints API when EndpointSlice is enabled. ([#4852](https://github.com/antrea-io/antrea/pull/4852), [@tnqn])
+- In Antrea Agent Service CIDR discovery, prevent headless Services from updating the discovered Service CIDR to avoid overwriting the default route of host network unexpectedly. ([#5008](https://github.com/antrea-io/antrea/pull/5008), [@hongliangl])
+- Use a random port when the UDP source port in a Traceflow is 0. ([#4963](https://github.com/antrea-io/antrea/pull/4963), [@gran-vmv])
+- Set default flag to 2 for TCP Traceflow to fix a Traceflow timeout issue when the flag is not provided. ([#4948](https://github.com/antrea-io/antrea/pull/4948), [@luolanzone])
+- Fix concurrent map write bug for LabelIdentity controller in Antrea Multi-cluster. ([#4994](https://github.com/antrea-io/antrea/pull/4994), [@Dyanngg])
+- Fix race condition between stale controller and ResourceImport reconcilers in Antrea Multi-cluster. ([#4853](https://github.com/antrea-io/antrea/pull/4853), [@Dyanngg])
+- Revert the change "Temporarily install Suricata from Ubuntu PPA instead of OISF PPA" to fix L7 NetworkPolicy failure. ([#4968](https://github.com/antrea-io/antrea/pull/4968), [@xliuxu])
```

Comment:
```suggestion
- Bump up Suricata to 6.0.12 to fix L7 NetworkPolicy issue. ([#4968](https://github.com/antrea-io/antrea/pull/4968), [@xliuxu])
```

Reply:
a L7 NetworkPolicy failure

the L7 NetworkPolicy issue